beta-src/src/state/game/game-api-slice.ts

```diff
@@ -374,23 +422,31 @@ const gameApiSlice = createSlice({
             },
           };
           setCommand(state, command, "mapCommands", "all");
-          if (currentOrders) {
-            const orderToUpdate = currentOrders.find(
-              (o) => o.unitID === currOrderUnitID,
-            );
-            if (orderToUpdate) {
-              updateOrdersMeta(state, {
-                [orderToUpdate.id]: {
-                  saved: false,
-                  update: {
-                    type: "Hold",
-                    toTerrID: null,
+
+          if (currentOrders?.length) {
+            phase === "Retreats"
+              ? updateOrdersMeta(state, {
+                  [order.orderID]: {
+                    saved: false,
+                    update: {
+                      type: "Disband",
+                      toTerrID: null,
+                    },
                   },
-                },
-              });
-            }
+                })
+              : updateOrdersMeta(state, {
+                  [order.orderID]: {
+                    saved: false,
+                    update: {
+                      type: "Hold",
+                      toTerrID: null,
+                    },
+                  },
+                });
           }
-          state.order.type = "hold";
+          phase === "Retreats"
+            ? (state.order.type = "hold")
+            : (state.order.type = "disband");
```

Comment:
Should be able to simplify to just the following.
`state.order.type = phase === "Retreats" ? "hold" : "disband";`